src/cli/onefuzz/api.py

```diff
@@ -811,9 +811,32 @@ def create(
         *,
         unmanaged: bool = False,
         arch: enums.Architecture = enums.Architecture.x86_64,
+        min_size: Optional[int] = 0,  # min size of pool
```

Comment:
A recent change to the CLI makes it such that we can  document arguments and have it show up in the CLI.

This is currently only used in the `template afl basic' component.

If you add the following as a function level pydoc:

```
        """
        Create a worker pool

        :param str name: Name of the worker-pool
        :param Optional[int] min_size: Specify the minimum size of the pool (when autoscaled)
        :param Optional[int] max_size: Specify the maximum size of the  pool (when autoscaled)
        """
```

Then arguments will show up like this:
```bash
$ onefuzz pools create --help |grep Name
  name                  Name of the worker-pool
$
```

Reply:
Note: this change isn't required to merge the pr, but for your FYI, given you are taking the time to explain the arguments.

Reply:
added it. this is amazing one.